src/streamlink/plugins/sportschau.py

```diff
@@ -1,4 +1,5 @@
 """
+$description Sporting channel live stream owned by Sportschau, a German sports media website.
```

Comment:
> Sporting channel

it is not a tv channel, just a sports magazine https://en.wikipedia.org/wiki/Sportschau

> owned by Sportschau

No, it is the same as ARD plugin